.github/workflows/o1vm-ci.yml

```diff
@@ -0,0 +1,96 @@
+name: o1vm CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "o1vm/**"
```

Comment:
When we do modify other parts of the code, like poly-commitment and co, we should also run it.

Reply:
> poly-commitment and co

@dannywillems can you please provide the full list of paths to include here?

Reply:
Or I can simply add all crates paths that are being imported into `o1vm`, is that ok?

Reply:
Can be ignored:
- arrabiata
- ivc
- folding
- mvpoly
- turshi
- circuit-construction
- book
- hasher

Reply:
So the dependencies are:
- curves
- groupmap
- kimchi
- internal-tracing
- msm
- o1vm
- poly-commitment
- poseidon
- srs
- utils

Reply:
```shell
cargo metadata --format-version 1 --no-deps | jq '.packages[] | select(.name == "o1vm") | .dependencies[] | select(.path != null) | .path'
```

Reply:
Oh so my approach gave me the following for o1vm workspace:

```
"/Users/serhii/projects/o1labs/proof-systems/folding"
"/Users/serhii/projects/o1labs/proof-systems/groupmap"
"/Users/serhii/projects/o1labs/proof-systems/kimchi"
"/Users/serhii/projects/o1labs/proof-systems/msm"
"/Users/serhii/projects/o1labs/proof-systems/curves"
"/Users/serhii/projects/o1labs/proof-systems/poseidon"
"/Users/serhii/projects/o1labs/proof-systems/poly-commitment"
```

But you also mentioned some that o1vm does not depend directly...